runtime-transaction/benches/process_compute_budget_instructions.rs

```diff
@@ -0,0 +1,141 @@
+#![feature(test)]
+extern crate test;
+use {
+    solana_runtime_transaction::instructions_processor::process_compute_budget_instructions,
+    solana_sdk::{
+        compute_budget::ComputeBudgetInstruction,
+        instruction::Instruction,
+        message::Message,
+        pubkey::Pubkey,
+        signature::Keypair,
+        signer::Signer,
+        system_instruction::{self},
+        transaction::{SanitizedTransaction, Transaction},
+    },
+    test::Bencher,
+};
+
+const NUM_TRANSACTIONS_PER_ITER: usize = 1024;
+
+fn build_sanitized_transaction(
+    payer_keypair: &Keypair,
+    instructions: &[Instruction],
+) -> SanitizedTransaction {
+    SanitizedTransaction::from_transaction_for_tests(Transaction::new_unsigned(Message::new(
+        instructions,
+        Some(&payer_keypair.pubkey()),
+    )))
+}
+
+#[bench]
+fn bench_process_compute_budget_instructions_empty(bencher: &mut Bencher) {
+    let tx = build_sanitized_transaction(&Keypair::new(), &[]);
+    bencher.iter(|| {
+        (0..NUM_TRANSACTIONS_PER_ITER).for_each(|_| {
+            assert!(
+                process_compute_budget_instructions(tx.message().program_instructions_iter())
+                    .is_ok()
+            )
+        })
+    });
+}
+
+#[bench]
+fn bench_process_compute_budget_instructions_non_builtins(bencher: &mut Bencher) {
+    let ixs: Vec<_> = (0..4)
+        .map(|_| {
+            Instruction::new_with_bincode(
+                Pubkey::create_program_address(&[b"", &[1]], &solana_sdk::bpf_loader::id())
+                    .unwrap(),
+                &0_u8,
+                vec![],
+            )
+        })
+        .collect();
+    assert_eq!(4, ixs.len());
```

Comment:
This assert seems odd to me, it couldn't possibly be anything else

Reply:
i forgot to remove this

Reply:
767c6f6e58a6942fd100e302404b92d1575d324b